### PR TITLE
Bleu branch

### PR DESCRIPTION
**Pull request by GitBleu**
New paragraph
Can you please integrate it?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,9 +4,9 @@
 
 ## Introduction to the British Social Attitudes Survey: R and SPSS Practical Exercises
 
-This repository contains the Quarto source files of the practical exercises for the Introduction to Social Attitudes Surveys UKDS Data Skills Module.
+This repository contains the Quarto source files of the practical exercises for the *Introduction to Social Attitudes Surveys* UKDS Data Skills Module.
```
Comment (PierreWal) [inline README.md:7]: I don't like this

File: README.md [hunk 2]
```diff
@@ -6,6 +6,14 @@
 
 This repository contains the Quarto source files of the practical exercises for the *Introduction to Social Attitudes Surveys* UKDS Data Skills Module.
 
+### The BSA
+
+A key aim of the BSA is to track the views and opinions of the public on national issues over time. Therefore, the BSA questionnaire is  designed to have several core questions that are repeated in most years. These cover different topics such as politics, welfare, poverty,  health, education, equalities, and employment.
```
Comment (PierreWal) [inline README.md:11]: I like this

File: README.md [hunk 3]
```diff
@@ -6,6 +6,14 @@
...
+
+In addition, the interview questionnaire consists of various  background and demographic questions. The rest of the questionnaire  includes a series of non-core questions (modules) on a number of social, political, economic and moral topics, which are included in the survey  less frequently.
```
Comment (PierreWal) [inline README.md:13]: This is useful too

File: README.md [hunk 4]
```diff
@@ -6,6 +6,14 @@
...
+
+The full questionnaire of the BSA survey used is available on the [documentation section](https://beta.ukdataservice.ac.uk/datacatalogue/series/series?id=200006#!/access-data) of the UK Data Service datasets for each survey year. [FAQs](https://beta.ukdataservice.ac.uk/datacatalogue/series/series?id=200006#!/faqs) are available.
```
Comment (PierreWal) [inline README.md]: This is not necessary. Can you please remove?

**Review by PierreWal — CHANGES_REQUESTED** [review]
See attached comments

File: README.md (hunk 4 above)
Comment (GitBleu) [reply README.md]: OK, I will remove

**Review by PierreWal — APPROVED** [review]
OK. Thanks

**Pull request merged**